CONTRIBUTING.md

```diff
@@ -208,7 +206,16 @@ Type can have following values: `WIP|feat|chore|test|docs|fix`.
 The `WIP` represent work in progress and it will not be merged, so please make sure to use one of 
 the specific format such as `feat|chore|test|doc|fix` if your PR needs to be merged with main 
 
+### Scope
+The scope should be the name of the npm package affected (as perceived by the person reading the changelog 
+generated from commit messages). Capitalize the first letter.
 
+The following is the list of supported scopes:
+
+* **core**
+* **platform**
+* **docs**
```

Comment:
We also have the scope `Core|Platform`

Reply:
in the Commitlint it colled multiple scopes
Commitlint supports multiple scopes.
Current delimiter options are:
- "/"
- "\"
- ","

Example: fix(core,platform): message

Reply:
I see, thanks for clarifying!

Reply:
if we are going to support multiple scopes then we need to point out what delimiters we should use and example how it could look like

Reply:
@dimamarksman updated
https://github.com/SAP/fundamental-ngx/pull/5414/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R152